Source/ThirdParty/Workers/draco-decoder-gltf.js

```diff
@@ -0,0 +1,27 @@
+define([], function() {
```

Comment:
Does this file belong in the `Workers` sub-directory?  This isn't a worker itself, right?

Reply:
No it's not a worker itself. When the file is in the `ThirdParty` directory, the module is still downloaded even when not using the webworker. Should I exclude it from `sourceFiles` in the gulpfile?

Reply:
I think excluding it in sourceFiles makes more sense, it looks like `pako_inflate.js`has the same problem (though it's not excluded, so we should probably add that too).  Write up an issue either way so that we can formalize the rules and update the coding guide for future reference.

Reply:
Opened https://github.com/AnalyticalGraphicsInc/cesium/issues/6298